vendor/modules.txt

```diff
@@ -8,10 +8,12 @@ github.com/sirupsen/logrus
 github.com/weppos/publicsuffix-go/publicsuffix
 # github.com/zmap/zcrypto v0.0.0-20190329181646-dff83107394d
 github.com/zmap/zcrypto/x509
+github.com/zmap/zcrypto/x509/ct
 github.com/zmap/zcrypto/x509/pkix
 github.com/zmap/zcrypto/json
-github.com/zmap/zcrypto/x509/ct
 # golang.org/x/crypto v0.0.0-20180904163835-0709b304e793
+golang.org/x/crypto/cryptobyte
```

Comment:
It seems reasonable to add deps on `/x/crypto/cryptobyte` and `/x/crypto/cryptobyte/asn1`. `zlint` was already carrying an indirect dependency on `/x/crypto` by way of `github.com/sirupsen/logrus` :+1: